proto/agglayer/node/v1/node_state.proto

```diff
@@ -54,20 +54,17 @@ enum GetNetworkStateErrorKind {
   // Unspecified error.
   GET_NETWORK_STATE_ERROR_KIND_UNSPECIFIED = 0;
 
-  // Missing latest settled certificate.
-  GET_NETWORK_STATE_ERROR_KIND_MISSING_LATEST_SETTLED_CERTIFICATE = 1;
-
-  // Missing latest pending certificate.
-  GET_NETWORK_STATE_ERROR_KIND_MISSING_LATEST_PENDING_CERTIFICATE = 2;
-
   // Invalid network ID/network not found.
-  GET_NETWORK_STATE_ERROR_KIND_INVALID_NETWORK_ID = 3;
+  GET_NETWORK_STATE_ERROR_KIND_INVALID_NETWORK_ID = 1;
```

Comment:
Should this be done in a more backwards compatible manner? I.e:
* mark the kinds 1 and 2 (and other removed kinds) as reserved
* maintain the same numeric mapping for `GET_NETWORK_STATE_ERROR_KIND_INVALID_NETWORK_ID` and `GET_NETWORK_STATE_ERROR_KIND_UNKNOWN_NETWORK_TYPE`
* allocate a fresh number for each newly introduced kind

Reply:
I'm not sure we're having backward compatibility concern as this feature is still in development

Reply:
> I'm not sure we're having backward compatibility concern as this feature is still in development

Ok, I was deceived by the fact it modifies an already merged proto definition.